### ISSUE TITLE
Add color to source code listing and disassembly

### ISSUE THREAD

**Issue by mattadev** [hugsy/gef#485]
Hi, please add the option for color syntax highlighting using pygmentize, for example, to highlight source code output (list command) and disassembly output (disas command).

**Comment by Grazfather** [comment]
We do not want any external dependencies. I suppose it could be done as a best effort.

**Comment by hugsy** [comment]
Agreed with @Grazfather , this kind of cosmetic feature would imply having a dependency which we don't want in the `gef` repository. 

@mattadev Feel free to submit a PR to `gef-extras`, some other people could be interested.

Cheers,

**Comment by hugsy** [comment]
Moving the issue to `gef-extras`
